### DISCUSSION TITLE
collection/gailan

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#129] [Announcements]
# collection/gailan

Java程序员进阶之路，小白的零基础Java教程，Java 集合框架（容器）体系结构

https://tobebetterjavaer.com/collection/gailan.html

**Comment by Knightzh** [comment]
看到队列里的LinkedList不能直接修改这句话直接懵逼了，因为List的代码里面明明就能set，然后仔细看了下，发现插入方式不同，作为list是add，作为队列是offer，还有poll。所以它作为这两种形式的时候用的函数是要独立的吗？不能交叉使用吗？一种形式不能用别的形式的函数吗？求二哥解答疑惑。

**Reply by 0319cc** [reply]
可以交叉使用啊，你别搞反了就行，本来这里面的函数大部分都是复用的
